packages/experience-toolkit/scripts/generate-api.js

```diff
@@ -0,0 +1,18 @@
+import path from 'node:path';
+
+import { generateApi } from 'swagger-typescript-api';
+
+const outputDirectory = path.resolve(import.meta.dirname, '../src/api');
+const openApiJsonUrl = 'https://auth.logto.app/api/.well-known/experience.openapi.json';
+
+generateApi({
+  name: 'experience-api.ts',
+  url: openApiJsonUrl,
+  output: outputDirectory,
+  httpClientType: 'fetch',
+  // Extract the response body type from the OpenAPI spec
+  extractResponseBody: true,
+  unwrapResponseData: true,
+  // This will group the routes by the first tag: experience and interaction
+  moduleNameFirstTag: true,
+});
```

Comment:
actually what i was expecting is to have an "official" experience API SDK in the main monorepo, which syncs with the version of `@logto/core`, and add as a dependency here.

Reply:
this is not a blocker, but somehow we should have that SDK, right?